### ISSUE TITLE
Move DATA section from Mojo::Util into a resources file

### ISSUE THREAD

**Issue by kraih** [mojolicious/mojo#1409]
Very simple task, we have other modules that do this already... like for the TLS certificates. So there is lots of examples.

**Comment by s1037989** [comment]
In wanting to use Mojo::File::path like Mojo::IOLoop::TLS, I'm having troubles with `use Mojo::File 'path';`.  Mojo::File -- through Collection/ByteSteam -- depends on Mojo::Util, so it seems that Mojo::File can't find what it needs from Mojo::Util yet because Mojo::Util isn't loaded yet.
Am I correct in perceiving that this would be a problem for this task?  Suggestions?  Should I not use Mojo::File and just find and open the `resources` file using core Perl functions?

**Comment by Grinnz** [comment]
You can use `require` to delay loading of another module so as not to create a circular import.

**Comment by Grinnz** [comment]
Example: https://metacpan.org/release/Mojolicious/source/lib/Mojo/Base.pm#L13-14

**Comment by s1037989** [comment]
Thanks, @Grinnz !

I'm on the latest commit, 4652927b
```
$ git log -1
commit 4652927b5b4fb3607dce667d013cb6611c6e49f9
Author: Sebastian Riedel <sri@cpan.org>
Date:   Thu Sep 12 00:33:19 2019 +0200

    Bump version
```

All tests pass:
```
$ prove -Ilib t/mojo/util.t 
t/mojo/util.t .. ok     
All tests successful.
Files=1, Tests=233,  1 wallclock secs ( 0.06 usr  0.01 sys +  0.39 cusr  0.01 csys =  0.47 CPU)
Result: PASS
```

Then I add Mojo::File:
```
$ git diff
diff --git a/lib/Mojo/Util.pm b/lib/Mojo/Util.pm
index 7171e36..b8f2958 100644
--- a/lib/Mojo/Util.pm
+++ b/lib/Mojo/Util.pm
@@ -36,6 +36,7 @@ use constant {
 
 # To generate a new HTML entity table run this command
 # perl examples/entities.pl
+require Mojo::File;
 my %ENTITIES;
 for my $line (split "\n", join('', <DATA>)) {
   next unless $line =~ /^(\S+)\s+U\+(\S+)(?:\s+U\+(\S+))?/;
```

Re-run the tests, and things break!
```
$ prove -Ilib t/mojo/util.t 
t/mojo/util.t .. "deprecated" is not exported by the Mojo::Util module
Can't continue after import errors at /tmp/mojo/lib/Mojo/Collection.pm line 9.
BEGIN failed--compilation aborted at /tmp/mojo/lib/Mojo/Collection.pm line 9.
Compilation failed in require at /tmp/mojo/lib/Mojo/File.pm line 21.
BEGIN failed--compilation aborted at /tmp/mojo/lib/Mojo/File.pm line 21.
Compilation failed in require at /tmp/mojo/lib/Mojo/Util.pm line 39.
Compilation failed in require at /tmp/mojo/lib/Mojo/Base.pm line 14.
Compilation failed in require at t/mojo/util.t line 1.
BEGIN failed--compilation aborted at t/mojo/util.t line 1.
```

**Comment by kraih** [comment]
Resolved by #1424.
